module_test.go

```diff
@@ -1,37 +1,16 @@
 package getter
 
 import (
-	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
 	"net/url"
-	"os"
 	"path/filepath"
-	"reflect"
-	"testing"
 
-	urlhelper "github.com/hashicorp/go-getter/helper/url"
+	urlhelper "github.com/hashicorp/go-getter/v2/helper/url"
 )
 
 const fixtureDir = "./testdata"
 
-func tempDir(t *testing.T) string {
-	dir, err := ioutil.TempDir("", "tf")
-	if err != nil {
-		t.Fatalf("err: %s", err)
-	}
-	if err := os.RemoveAll(dir); err != nil {
-		t.Fatalf("err: %s", err)
-	}
-
-	return dir
-}
-
-func tempTestFile(t *testing.T) string {
-	dir := tempDir(t)
-	return filepath.Join(dir, "foo")
-}
-
 func testModule(n string) string {
```

Comment:
Can this go in `helper/testing/utils.go` as well or do you want to add this later to the FileGetter and HttpGetter separately? Since only these two use what's left.

Reply:
I want the FileGetter and HttpGetter to remain in the main pkg and since they are the only ones using it I am leaving this here.